regexes/client/libraries.yml

```diff
@@ -357,3 +367,8 @@
 - regex: 'harbor-registry-client'
   name: 'Harbor registry client'
   version: ''
+
+# Generic detections
+- regex: 'github'
```

Comment:
@sgiehl can you look at this condition is it acceptable to enter a common name for undefined libraries into the detector?

Reply:
I wouldn't add generic libraries. They might also be hard to differ from a bot :thinking: